backend/controller/controller.go

```diff
@@ -169,12 +169,6 @@ func New(ctx context.Context, db *dal.DAL, config Config, runnerScaling scaling.
 	return svc, nil
 }
 
-type HTTPResponse struct {
-	Status  int
-	Headers map[string][]string
-	Body    []byte
```

Comment:
I think if you make this `Body json.RawMessage` it would automatically do most of what you want, and you could avoid all the digging around in the responseMap below.

Reply:
That's soooo much better, and fixes a bug! :)